lib/std/crypto/25519/field.zig

```diff
@@ -0,0 +1,327 @@
+const std = @import("std");
+const readIntLittle = std.mem.readIntLittle;
+const writeIntLittle = std.mem.writeIntLittle;
+
+pub const Fe = struct {
+    limbs: [5]u64,
+
+    const MASK51: u64 = 0x7ffffffffffff;
+
+    pub inline fn zero() Fe {
```

Comment:
These don't need to be inline;
hell, they don't even need to be functions as far as I can see? just have constants.

Reply:
Everything has been converted to constants, thanks!